### ISSUE TITLE
Add more of the supported operations to net topology suite docs

### ISSUE THREAD

**Issue by oshawaConnectionJBA** [npgsql/doc#59]
Checking out [the docs on spatial data for npgsql](http://www.npgsql.org/efcore/mapping/nts.html#operation-translation) only a sample of the operations supported by this library are written - [far more are supported by this library](https://docs.microsoft.com/en-us/ef/core/modeling/spatial#translated-operations)

I think the docs should be updated to cover these operations, or the docs should point to the MS docs (as the top of the page suggests).

**Comment by roji** [comment]
The EF Core page with spatial support across providers is being removed (see https://github.com/dotnet/EntityFramework.Docs/issues/2611), each provider has its own page detailing support. The Npgsql page is [here](http://www.npgsql.org/efcore/mapping/nts.html#operation-translation), and indeed needs to be brought up to date.

Would you like to contribute a PR doing this?

**Comment by oshawaConnectionJBA** [comment]
Hi @roji, yeah sure! I will take a look this weekend from my personal account (oshawa-connection). 
[Its just a case of editing this file right?](https://github.com/npgsql/doc/blob/main/conceptual/EFCore.PG/mapping/nts.md)

**Comment by roji** [comment]
Yep, that's the file. You can just submit a PR against the main branch of the docs repo with the appropriate changes.
